adding docstring

SimardeepKaur opened the pull request:
Docstring for theme_ubc

SimardeepKaur committed:
adding docstring

reikookamoto reviewed (commented):
Hi Simardeep, sorry if this causes you any inconvenience but I'm choosing not to accept this pull request for a few reasons.

According to https://r-pkgs.org/whole-game.html#document, we should create the documentation backbone by doing **Code > Insert roxygen skeleton** in RStudio.  For that reason, this documentation is structured differently than the one in the example (e.g. missing @export). Moreover, I thought we agreed to follow Evelyn's documentation style during the meeting.

Is it possible for you to make these two changes? Please let me know if you disagree with anything I said above. Thanks!

SimardeepKaur commented:
I am sorry for the confusion but I added the documentation using Code > Insert roxygen skeleton and removed @export intentionally as there was no proper use for this as now.
Maybe I misinterpreted using the earlier discussed template because I found it quite generic and hence made changes accordingly.
It will be great if you go ahead with what you have in your mind, I would be happy to change it accordingly.

evelynmoorhouse commented:
Hi all, 

I'm just about to add my docstring. So we can essentially use that exact file (with the exception of the extra U of T argument for `colour_palette`), and just change the names of the functions throughout. Hope this helps!

SimardeepKaur committed:
adding theme

SimardeepKaur committed:
added theme
